namui-cli/src/util/get_namui_bundle_list/lexicon/namui_bundle_list.rs

```diff
@@ -0,0 +1,45 @@
+use super::{ExcludeOperation, IncludeOperation};
+use std::{collections::HashMap, path::PathBuf};
+
+type SrcDestPathMap = HashMap<PathBuf, PathBuf>;
+
+#[derive(PartialEq, Eq, Debug)]
+pub struct NamuiBundleList {
```

Comment:
I suggest rename it again... to "NamuiBundleMenifest" or something like that. How about this?
It's not actually the list of bundle, it is kind of information what kind of bundle namui-cli should take. So it should have prefer name for the purpose.

Reply:
Aaah that looks good